src/figma-json.ts

```diff
@@ -925,7 +925,7 @@ export interface PageNode extends BaseNodeMixin, ChildrenMixin, ExportMixin {
   readonly type: "PAGE";
 
   guides: ReadonlyArray<Guide>;
-  selection: ReadonlyArray<SceneNode>;
+  //selection: ReadonlyArray<SceneNode>;
```

Comment:
Maybe not necessary in this case but `CONVERSION` comment with rationale for why we're commenting this out it useful for future visitors